example/.eslintrc

```diff
@@ -14,6 +18,7 @@
     "@typescript-eslint/no-non-null-assertion": "off",
     "@typescript-eslint/no-empty-function": "off",
     "react/jsx-uses-react": "error",
-    "react/react-in-jsx-scope": "error"
+    "react/react-in-jsx-scope": "error",
+    "reanimated/js-function-in-worklet": 0
```

Comment:
Here the rule is disabled, once the useDerivedValue hook is renamed, we can reactivate the rule